packages/typespec-test/test/batch_modular/generated/typespec-ts/src/BatchClient.ts

```diff
@@ -1098,31 +1234,42 @@ export class BatchClient {
    * support to aid in debugging issues with the Batch service.
    */
   uploadNodeLogs(
+    apiVersion: string,
     poolId: string,
     nodeId: string,
     body: UploadBatchServiceLogsOptions,
     options: UploadNodeLogsOptions = { requestOptions: {} },
   ): Promise<UploadBatchServiceLogsResult> {
-    return uploadNodeLogs(this._client, poolId, nodeId, body, options);
+    return uploadNodeLogs(
+      this._client,
+      apiVersion,
+      poolId,
+      nodeId,
+      body,
+      options,
+    );
   }
 
   /** Lists the Compute Nodes in the specified Pool. */
   listNodes(
+    apiVersion: string,
     poolId: string,
     options: ListNodesOptions = { requestOptions: {} },
   ): PagedAsyncIterableIterator<BatchNode> {
-    return listNodes(this._client, poolId, options);
+    return listNodes(this._client, apiVersion, poolId, options);
   }
 
   /** Gets information about the specified Compute Node Extension. */
   getNodeExtension(
+    apiVersion: string,
```

Comment:
I notice that this is the case RLC has default value so skiped to generate query param. 
Do we need to following same rule in RLC to promot apiVersion to client level?

Another thing is I notice this parameter is not passed to RLC even though it is required one now.